compiler/modules/nback/basic.bal

```diff
@@ -222,7 +222,8 @@ function buildFunctionDecl(Scaffold scaffold, bir:ExternalSymbol symbol, bir:Fun
     }
 }
 
-function buildFunctionSignature(bir:FunctionSignature signature, err:Location loc) returns llvm:FunctionType|BuildError {
+// pr-todo: remove loc param
+function buildFunctionSignature(bir:FunctionSignature signature, err:Location? loc = ()) returns llvm:FunctionType|BuildError {
```

Comment:
I am planing to remove  `err:Location loc`.
@heshanpadmasiri

Reply:
Move this into build.bal since it's going to be used by init.bal.